snowfakery_samples/EDA/eda_objects.yml

```diff
@@ -95,15 +103,14 @@
     #   reference: BossContact # you need to have defined this Contact elsewhere
     hed__PreferredPhone__c: "Mobile Phone"
     Email:
-      fake: email
+      fake: Email
     Title:
       fake.text:
         max_nb_chars: 100
     Department:
       fake.text:
         max_nb_chars: 80
-    AssistantName:
-      fake: name
+    AssistantName: "${{fake.FirstName}} ${{fake.LastName}}"
```

Comment:
Why did you decide to use First + Last instead of Fake Name?

Reply:
Fake Name was giving me prefixes like Dr. and Ms., which didn't feel right for this field:

<img width="1240" alt="Screen Shot 2021-06-10 at 1 59 01 PM" src="https://user-images.githubusercontent.com/206762/121574277-18c99780-c9f4-11eb-8526-4b7cd62861be.png">

Reply:
Seems reasonable, thanks!